roles/postgres/tasks/main.yml

```diff
@@ -62,20 +62,35 @@
     ansible_ssh_pipelining: yes
   block:
     - name: Create necessary databases
-      postgresql_db:
+      community.postgresql.postgresql_db:
         name: "{{ item }}"
       loop:
         - "{{ repo_db_name }}"
         - "{{ sync_db_name }}"
 
-    - name: Set users privileges on databases
-      postgresql_user:
+    # https://dba.stackexchange.com/a/35317/5179
```

Comment:
I would rather put that in the JIRA's & PR comments rather than the code itself. So people can always track back to it using git blame or github PR.
Having it in the code can make sense if it allows pointing something that needs to later be changed but here that's just how things have to work anyway